spec/integ/crypto/cross-signing.spec.ts

```diff
@@ -347,6 +347,81 @@ describe.each(Object.entries(CRYPTO_BACKENDS))("cross-signing (%s)", (backend: s
+                            content: {
+                                algorithm: "m.secret_storage.v1.aes-hmac-sha2",
+                                passphrase: {
+                                    algorithm: "m.pbkdf2",
+                                    iterations: 500000,
+                                    salt: "W7O4C37CwdK5o6tvZtc7FYBYLBU5XJfN",
+                                },
+                                iv: "LGQi6b/F4qQCkXt2gLVwuw==",
+                                mac: "KdZHA+WyHxVSXuHeBD7LDioAbiw7M3ylhl5edGqyXNU=",
+                            },
+                        },
+                        {
+                            type: "m.secret_storage.default_key",
+                            content: {
+                                key: "lGZnUDzNKXGXR083k8JVKAtGSiBj5Dv1",
+                            },
+                        },
+                        {
+                            type: "m.cross_signing.master",
+                            content: {
+                                encrypted: {
+                                    lGZnUDzNKXGXR083k8JVKAtGSiBj5Dv1: {
+                                        iv: "6L66/iD9KZg6DVdOMfEXgQ==",
+                                        ciphertext: "S42Ez1Xpm29SWPb9gQAKzV4WkfX9B5xU+CXG4TzdsYi1bVEXQ5Eps+z6Pw==",
+                                        mac: "iwyUtX2wdGKF+ljiMa9Le0Ub0ci7qQLWXAsN8Qp6JDE=",
+                                    },
+                                },
+                            },
+                        },
+                        {
+                            type: "m.cross_signing.user_signing",
+                            content: {
+                                encrypted: {
+                                    lGZnUDzNKXGXR083k8JVKAtGSiBj5Dv1: {
+                                        iv: "MBt9eQi5O8UOPV7HRbr3rA==",
+                                        ciphertext: "eDRiI9nXa4kSZxwuW/j+JYlvSYqGbauffnlv3+Cyz6xbkWvZu3kcirhK8Q==",
+                                        mac: "f47JSj6piHEeaavTa03wqWouuNfhIHuSFX6HGatmu0k=",
+                                    },
+                                },
+                            },
+                        },
+                        {
+                            type: "m.cross_signing.self_signing",
+                            content: {
+                                encrypted: {
+                                    lGZnUDzNKXGXR083k8JVKAtGSiBj5Dv1: {
+                                        iv: "6YJA6EYa72xbttCd981DZw==",
+                                        ciphertext: "1ZVy5fWqLPCVsJPvi2L/Pw9uS1D02MkeZ8zNwh7knA8hveOMO2PBslf8EQ==",
+                                        mac: "2EXorwWE5unhdsLx+IO7hxhZwolHyKoLV2ZKaRJoBlQ=",
+                                    },
+                                },
+                            },
+                        },
+                    ],
```

Comment:
could you write some comments about what exactly these events are and where they came from? Are they supposed to match the keys used in `test-data`, or are they deliberately different? In other words, do we expect the client to be able to decrypt them or not?

Reply:
Udpated

Reply:
This is much clearer now, thanks!